### ISSUE TITLE
hoveredOutlierIndex returns -1 every time

### ISSUE THREAD

**Issue by SorataYuu** [sgratzl/chartjs-chart-boxplot#17]
When I have a box plot, and mouse over an outlier. The tooltip context returns `hoveredOutlierIndex` as `-1` despite being hovered over an outlier. 

I tried to access it using this.
```
chart_config.options.interaction = {
            mode: 'index'
        };

        chart_config.options.plugins.tooltip.callbacks.footer = (tooltip_items) => {
            console.log(tooltip_items);
        }
```

**Comment by sgratzl** [comment]
can you provide a sample codepen, please

**Comment by SorataYuu** [comment]
I've made a codepen that prints the `hoveredOutlierIndex` on the tooltip:
https://codepen.io/soratayuu/pen/MWOrNxw

**Comment by sgratzl** [comment]
take a look at https://codepen.io/sgratzl/pen/bGYLqWK?editors=0010

I just released v3.7.0 which adapt for chart.js 3.7
